docs/walkthrough.md

```diff
@@ -0,0 +1,500 @@
+Walkthrough
+===========
+
+This walkthrough will go over the basics of setting up the Prometheus
+adapter on your cluster and configuring an autoscaler to use application
+metrics sourced from the adapter.
+
+Prerequisites
+-------------
+
+### Cluster Configuration ###
+
+Before getting started, ensure that the main components of your
+cluster are configured for autoscaling on custom metrics.  As of
+Kubernetes 1.7, this requires enabling the aggregation layer on the API
+server and configuring the controller manager to use the metrics APIs via
```

Comment:
I provide a link below ("Detailed instructions can be found in the Kubernetes documentation...").  I can provide a link to the `kubeadm` docs as well, if there is one.  I don't use `kubeadm`, so I wasn't familiar with the config required.